src/test/java/uk/gov/hmcts/sptribs/systemupdate/event/SystemMigrateCaseFlagsTest.java

```diff
@@ -44,6 +53,7 @@ void shouldAddConfigurationToConfigBuilderWithToggleOn() {
     void shouldAddConfigurationToConfigBuilderWithToggleOff() {
```

Comment:
method name should be `shouldNotAddConfigurationToConfigBuilderWithToggleOff`

Reply:
Same for `SystemMigrateCaseLinksTest. shouldAddConfigurationToConfigBuilderWithToggleOff()` -> `SystemMigrateCaseLinksTest. shouldNotAddConfigurationToConfigBuilderWithToggleOff()`